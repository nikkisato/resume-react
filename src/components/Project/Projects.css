.projects{
    width: 200px;
}