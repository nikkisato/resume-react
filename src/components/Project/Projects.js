import React from 'react';
import data from '../resume2';
import styles from './Projects.css';

export default function Education() {
  return (
    <div className={styles.projects}>
      <h1>Projects</h1>
      <h2>{data.Projects[0].title}</h2>
      <p><i>{data.Projects[0].tech}</i></p>
      <p>{data.Projects[0].description}</p>

      <h2>{data.Projects[1].title}</h2>
      <p><i>{data.Projects[1].tech}</i></p>
      <p>{data.Projects[1].description}</p>


      <h2>{data.Projects[2].title}</h2>
      <p><i>{data.Projects[2].tech}</i></p>
      <p>{data.Projects[2].description}</p>
    </div>
  );
}
