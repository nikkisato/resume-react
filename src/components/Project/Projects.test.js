import React from 'react';
import { shallow } from 'enzyme';
import Projects from './Projects';

describe('Projects component', () => {
  it('renders Projects', () => {
    const wrapper = shallow(<Projects />);
    expect(wrapper).toMatchSnapshot();
  });
});
