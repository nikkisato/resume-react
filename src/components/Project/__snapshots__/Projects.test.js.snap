// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component renders Projects 1`] = `
<div
  className="projects"
>
  <h1>
    Projects
  </h1>
  <h2>
    CovrLettr
  </h2>
  <p>
    <i>
      Express, MongoDB, Node.js, Javascript
    </i>
  </p>
  <p>
    Writing Cover Letters for people who are entering the software developer field. The program will ask a series of questions, take in user input, and output a customized cover letter. Currently a Backend Project for the moment.
  </p>
  <h2>
    Nag Me
  </h2>
  <p>
    <i>
      PostgresSQL, Express, Node.js, Javascript
    </i>
  </p>
  <p>
    NagMe is the super proactive task and reminder app, that uses Pushover (push notifications) to send notifications to your phone. Users can add, update, delete Nags
  </p>
  <h2>
    Guess Who
  </h2>
  <p>
    <i>
      Vanilla Javascript, HTML/CSS
    </i>
  </p>
  <p>
    The classic Guess Who board game is now online. Used a different set of characters (aliens) to make a different experience of the game. Used local storage to persist user data and leaderboards.
  </p>
</div>
`;
